db/migrate/20200213205538_add_initial_facets.rb

```diff
@@ -0,0 +1,61 @@
+class AddInitialFacets < Mongoid::Migration
```

Comment:
A couple things about these initial facets: technically, we are referring to the convention as "alexandria_convention" - that's the name of the table in BigQuery, so we should be consistent in that.  Also, the current version is 1.1.3, not 1.0.  We should be using the JSON schema to manage all of these values:  https://github.com/broadinstitute/scp-ingest-pipeline/blob/master/tests/data/AMC_v1.1.3.json.